src/components/utils/http.js

```diff
@@ -13,7 +13,17 @@ const objectToFormData = (object) =>
     return formData;
   }, new FormData())
 
-const handleError = (message, callback) => {
+const errorMessage = (status) => {
+  if (status === 401 || status === 403) return 'unauthorized'
```

Comment:
The server response should include a useful message as well as a status. I think we should log out the message from the server to make it easier for developers to debug